public/docs/_examples/dependency-injection/dart/lib/heroes/mock_heroes.dart

```diff
@@ -0,0 +1,15 @@
+// #docregion
+import "hero.dart";
+
+List<Hero> HEROES = [
+  new Hero()..id = 11..isSecret = false..name = "Mr. Nice",
```

Comment:
nop, it put every field in a new line, do you want me to change to multi-line format?